### DISCUSSION TITLE
Forked Dependencies?

### DISCUSSION THREAD

**Discussion by rohnyn** [jayofelony/pwnagotchi#216] [Q&A]
Why does the build image pull from a forked bettercap?

**Comment by jayofelony** [comment]
I answered this already..

**Reply by rohnyn** [reply]
Bettercap isn't abandoned... In fact, it's several releases ahead of the forked repo pwnagotchi is sourcing from. What functionality was modified?

https://github.com/bettercap/bettercap

**Reply by rohnyn** [reply]
for viewers reference this is follow-up discussion thread from https://github.com/jayofelony/pwnagotchi/issues/215

**Reply by rohnyn** [reply]
@jayofelony This shouldn't be marked as resolved yet. I think it may be good practice to cut back to upstream unless there's a valid reason to use a fork.

**Reply by jayofelony** [reply]
Bettercap only recently got updated, it had not been for the 2 years before that.

**Reply by jayofelony** [reply]
I'm also not using ansible anymore, if there will be a new version it will use the latest bettercap.
